superset-frontend/plugins/plugin-chart-echarts/src/controls.tsx

```diff
@@ -136,10 +136,29 @@ const onlyTotalControl = {
   },
 };
 
+const percentageThresholdControl = {
+  name: 'percentage_threshold',
+  config: {
+    type: 'TextControl',
+    label: t('Percentage threshold'),
+    renderTrigger: true,
+    isFloat: true,
+    default: 5,
```

Comment:
This doesn't be in line with `DEFAULT_FORM_DATA.percentageThreshold` in `types.ts`, which is set to `0`. Perhaps we should import the defaults and use that here, something like
```javascript
import { DEFAULT_LEGEND_FORM_DATA } from './types';

const percentageThresholdControl = {
  name: ...,
  config: {
    ...,
    default: DEFAULT_FORM_DATA.percentageThreshold,
    ....
  },
}
```

Reply:
Ah yes, that is a bit confusing. What you got here makes a lot of sense. I will change it to this and push it up!